djangocms_versioning/templates/djangocms_versioning/admin/revert_icon.html

```diff
@@ -0,0 +1 @@
+{% load static i18n %}<a {% if disable_revert_link %}class="btn cms-versioning-action-btn inactive" title="{% trans 'Draft already exists, revert action not possible' %}"{% else %}class="btn cms-versioning-action-btn js-versioning-action" href="{{ revert_url }}" title="{% trans 'Revert' %}"{% endif %}> <img src="{% static 'djangocms_versioning/svg/undo.svg' %}"></a>
```

Comment:
You're passing that `disabled` variable to the context, but it's not being used

Reply:
Also, you can split the tag into multiple lines so it's more readable (and then use `{% spaceless %}` around it, if you want to keep the output in one line)